.github/workflows/sdist.yml

```diff
@@ -59,12 +59,12 @@ jobs:
         name: ${{matrix.python-version}}-sdist.gz
         path: dist/*.gz
 
-    - uses: conda-incubator/setup-miniconda@v2.1.1
+    - name: Set up Conda
+      uses: ./.github/actions/setup-conda
       with:
-        activate-environment: pandas-sdist
-        channels: conda-forge
-        python-version: '${{ matrix.python-version }}'
-        condarc-file: ci/condarc.yml
+        environment-file: ""
```

Comment:
Does this run correctly if empty string is provided?

Reply:
Yes it is the default value, see https://github.com/conda-incubator/setup-miniconda/blob/e81abac10ce2c37423b54eae5af93aa3b4d3475c/action.yml#L78. I actually checked that `sdist` works in an earlier version of the PR by uncommenting the `if`.